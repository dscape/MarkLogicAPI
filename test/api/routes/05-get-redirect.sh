#!/bin/sh
curl -vs localhost:6984/_download 2>&1 | sed -e 's/^\*.*//' | grep '<'
