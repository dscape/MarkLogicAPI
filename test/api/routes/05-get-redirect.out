< HTTP/1.1 302 Found
< Location: http://developer.marklogic.com/products
< Server: MarkLogic
< Content-Length: 0
< Connection: close
< 
